_includes/footer.html

```diff
@@ -21,16 +21,22 @@ <h4><strong>{{ site.owner }}</strong>
         <br>
         <ul class="list-inline">
           <li>
-            <a href="#"><i class="fa fa-dribbble fa-fw fa-3x"></i></a>
+            <a href="#"><i class="fas fa-arrow-circle-up fa-fw fa-3x"></i></a> <!-- previously used fab fa-dribbble-->
           </li>
           <li>
-            <a href="https://facebook.com/{{ site.facebook_username }}"><i class="fa fa-facebook fa-fw fa-3x"></i></a>
+            <a href="https://facebook.com/{{ site.facebook_username }}"><i class="fab fa-facebook fa-fw fa-3x"></i></a>
           </li>
           <li>
-            <a href="https://twitter.com/{{ site.twitter_username }}"><i class="fa fa-twitter fa-fw fa-3x"></i></a>
+            <a href="https://twitter.com/{{ site.twitter_username }}"><i class="fab fa-twitter fa-fw fa-3x"></i></a>
           </li>
           <li>
-            <a href="https://github.com/{{ site.github_username }}"><i class="fa fa-github fa-fw fa-3x"></i></a>
+            <a href="https://github.com/{{ site.github_username }}"><i class="fab fa-github fa-fw fa-3x"></i></a>
+          </li>
+          <li>
+            <a href="https://t.me/osmUA/{{ site.telegram_username }}"><i class="fab fa-telegram fa-fw fa-3x"></i></a>
+          </li>
+          <li>
+            <a href="https://osmukraine.slack.com/{{ site.slack_username }}"><i class="fab fa-slack fa-fw fa-3x"></i></a>
```

Comment:
Треба винести `{{ site.slack_username }}` з href="…" через формування некоректного посилання.  `osmukraine.slack.com` краще замінити на переадресацію на посилання-запрошення через `bit.ly`

```suggestion
            <a href="https://bit.ly/SlackOsmUa/" title="{{ site.slack_username }}"><i class="fab fa-slack fa-fw fa-3x"></i></a>
```

Reply:
Можна, тільки без слеша в кінці: "https://bit.ly/SlackOsmUa". Зі слешем викидає на 404.
А чому краще через bit.ly, поясни, будь ласка? Я зазвичай не люблю ставити на сайт довготривалі посилання через скорочувач - відвідувач може ставитися до такого з підозрою (принаймні, я так ставлюся).